src/Honeybee_FileExplorer.py

```diff
@@ -0,0 +1,49 @@
+#Developed by: Sarith Subramniam, Penn State
+#sarith@sarith.in
+#Demo: https://youtu.be/KR5tRepXgRA
+"""
+    
+    Locate a file/directory in windows explorer
+    
+    Args:
+        Destination: File path or Directory path
```

Comment:
Input/output names should be camelCase, and should have "_" as needed. In your case since both inputs are required they need "_" before the input name. _destination, and _open